client/blocks/signup-form/index.jsx

```diff
@@ -965,6 +968,18 @@ class SignupForm extends Component {
 			);
 		}
 
+		if ( this.props.currentUser ) {
+			return (
+				<ContinueAsUser
+					redirectPath={ this.props.redirectToAfterLoginUrl }
+					onChangeAccount={ () => {
+						// todo: Add track this event!
```

Comment:
Are we planning to address this TODO in this PR?

Reply:
Done we should be tracking things now.